common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -119,6 +119,7 @@ export type ICommittedProposal = {
 // @public
 export interface IConnect {
     client: IClient;
+    correlationProperties?: Record<string, unknown>;
```

Comment:
I'm not sure we want a collection of properties. As that suggests it's very extensible mechanism that can be used for a lot of things. I think we should push people toward supportedFeatures as such mechanism.
For logging, I prefer much simpler and limited "agent string" approach, i.e. it's just a string that service formats however it wants and we log it as is, treating as black box.

The only reason I see we might want to go with property set is ability to log individual fields in telemetry to be able to query based on individual things. That said, a bunch of parsing capabilities exists in Kusto, so if agent string is well formatted, we can still do it even if it's single string.

See here: https://www.w3schools.com/jsref/prop_nav_useragent.asp
Here is its value for me:

'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.54 Safari/537.36 Edg/95.0.1020.30'

Reply:
Changed it to string.